rust/cymbal/src/frames/mod.rs

```diff
@@ -15,17 +17,21 @@ pub mod resolver;
 // We consume a huge variety of differently shaped stack frames, which we have special-case
 // transformation for, to produce a single, unified representation of a frame.
 #[derive(Debug, Deserialize, Serialize, Clone)]
-#[serde(untagged)]
+#[serde(tag = "platform")]
 pub enum RawFrame {
+    #[serde(rename = "python")]
+    Python(RawPythonFrame),
+    #[serde(rename = "web:javascript")]
     JavaScript(RawJSFrame),
 }
 
 impl RawFrame {
     pub async fn resolve(&self, team_id: i32, catalog: &Catalog) -> Result<Frame, UnhandledError> {
-        let RawFrame::JavaScript(raw) = self;
-
         let frame_resolve_time = common_metrics::timing_guard(PER_FRAME_TIME, &[]);
-        let res = raw.resolve(team_id, catalog).await;
+        let (res, lang_tag) = match self {
+            RawFrame::JavaScript(frame) => (frame.resolve(team_id, catalog).await, "js"),
+            RawFrame::Python(frame) => (Ok(frame.into()), "python"),
```

Comment:
Feel like we should stick with one or the other for consistency
```suggestion
            RawFrame::JavaScript(frame) => (frame.resolve(team_id, catalog).await, "javascript"),
            RawFrame::Python(frame) => (Ok(frame.into()), "python"),
```
```suggestion
            RawFrame::JavaScript(frame) => (frame.resolve(team_id, catalog).await, "js"),
            RawFrame::Python(frame) => (Ok(frame.into()), "py"),
```

Reply:
I didn't think about it very much because it's just for metrics, but yeah, lets go with the full names